xml/nsx-deploy-cloud.xml

```diff
@@ -13,10 +13,9 @@
   variable before running <filename>site.yml</filename>.
  </para>
  <screen>&prompt.ardana;cd ~/openstack/ardana/ansible
- &prompt.ardana;ansible-playbook -i hosts/localhost ready-deployment.yml
- &prompt.ardana;cd ~/scratch/ansible/next/ardana/ansible
- &prompt.ardana;export ARDANA_USER_PASSWORD_ENCRYPT_KEY=<replaceable>KEY_USED_TO_ENCRYPT_THE_VCENTER_AND_NSX_MGR_PASSWORDS</replaceable>
- &prompt.ardana;ansible-playbook -i hosts/verb_hosts site.yml
- &prompt.ardana;ansible-playbook -i hosts/verb_hosts
- ardana-cloud-configure.yml</screen>
+&prompt.ardana;ansible-playbook -i hosts/localhost ready-deployment.yml
+&prompt.ardana;cd ~/scratch/ansible/next/ardana/ansible
+&prompt.ardana;export ARDANA_USER_PASSWORD_ENCRYPT_KEY=<replaceable>KEY_USED_TO_ENCRYPT_THE_VCENTER_AND_NSX_MGR_PASSWORDS</replaceable>
```

Comment:
The replaceable makes this line longer than 80 characters... It is usually better to use short replaceables and explain them below the screen. (This screen is rather too long and unexplained anyway.)